### ISSUE TITLE
v2.6.2: Cannot create webserver: TypeError: Cannot read properties of undefined (reading 'createServerAsync')

### ISSUE THREAD

**Issue by hsteinme** [ioBroker/ioBroker.simple-api#195]
Instance stops directly after start with log entry "Cannot create webserver: TypeError: Cannot read properties of undefined (reading 'createServerAsync')"

Log says:
```
simple-api.0 | 2024-05-20 12:18:02.224 | info | Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason
simple-api.0 | 2024-05-20 12:18:02.223 | debug | Plugin sentry destroyed
simple-api.0 | 2024-05-20 12:18:02.221 | error | Cannot create webserver: TypeError: Cannot read properties of undefined (reading 'createServerAsync')
simple-api.0 | 2024-05-20 12:18:02.196 | info | starting. Version 2.7.2 in /opt/iobroker/node_modules/iobroker.simple-api, node: v18.20.2, js-controller: 5.0.19
simple-api.0 | 2024-05-20 12:18:01.564 | debug | Plugin sentry Initialize Plugin (enabled=true)
```

![image](https://github.com/ioBroker/ioBroker.simple-api/assets/33778564/8769f1c8-5294-4907-bc08-7727c2a13275)

Adapter version 2.6.2

```
======================= SUMMARY =======================
                        v.2024-04-21


   Static hostname: RaspberryPi
         Icon name: computer
  Operating System: Raspbian GNU/Linux 10 (buster)
            Kernel: Linux 5.10.103-v7l+
      Architecture: arm

Installation:           native
Kernel:                 armv7l
Userland:               32 bit
Timezone:               Europe/Berlin (CEST, +0200)
User-ID:                1000
Display-Server:         false
Boot Target:            multi-user.target

Pending OS-Updates:     0
Pending iob updates:    0

Nodejs-Installation:
/usr/bin/nodejs         v18.20.2
/usr/bin/node           v18.20.2
/usr/bin/npm            10.5.0
/usr/bin/npx            10.5.0
/usr/bin/corepack       0.25.2

Recommended versions are nodejs 18.20.2 and npm 10.5.0
Your nodejs installation is correct

MEMORY:
              total        used        free      shared  buff/cache   available
Mem:           3.8G        2.1G        847M        9.0M        879M        1.6G
Swap:           99M          0B         99M
Total:         3.9G        2.1G        947M

Active iob-Instances:   39
Active repo(s): Stable (default), Beta (latest)

ioBroker Core:          js-controller           5.0.19
                        admin                   6.17.7

ioBroker Status:        iobroker is running on this host.


Objects type: jsonl
States  type: jsonl

Status admin and web instance:
+ system.adapter.admin.0                  : admin                 : RaspberryPi                              -  enabled, port: 8081, bind: 0.0.0.0, run as: admin
+ system.adapter.web.0                    : web                   : RaspberryPi                              -  enabled, port: 8082, bind: 0.0.0.0, run as: admin

Objects:                21997
States:                 20770

Size of iob-Database:

26M     /opt/iobroker/iobroker-data/objects.jsonl
12M     /opt/iobroker/iobroker-data/objects.json.migrated
12M     /opt/iobroker/iobroker-data/objects.json.bak.migrated
12K     /opt/iobroker/iobroker-data/objects.json.old
17M     /opt/iobroker/iobroker-data/states.jsonl
4.1M    /opt/iobroker/iobroker-data/states.json.migrated
4.1M    /opt/iobroker/iobroker-data/states.json.bak.migrated
3.5M    /opt/iobroker/iobroker-data/states.json.old



=================== END OF SUMMARY ====================

```

**Comment by krobipd** [comment]
same problem here
Cannot create webserver: TypeError: Cannot read properties of undefined (reading 'createServerAsync')
<img width="930" alt="Bildschirmfoto 2024-05-20 um 21 31 52" src="https://github.com/ioBroker/ioBroker.simple-api/assets/27220902/a98a31f5-2850-4117-86a1-9adfd6ff216b">

**Comment by P4tiron** [comment]
I got exactly the same issue 2 days ago.
the Log says:
![image](https://github.com/ioBroker/ioBroker.simple-api/assets/170519046/4168d9a2-3487-41c1-8cbf-b487554c03ab)
Adapter Version 2.7.2
Node.js Version 20.13.1
NPM Version 10.8.0

![image](https://github.com/ioBroker/ioBroker.simple-api/assets/170519046/bb950057-97c3-4d54-859b-6449056780f0)

**Comment by Totte23** [comment]
Same with me:

<html>
<body>

simple-api.0 | 2024-05-22 21:09:55.138 | error | Cannot create webserver: TypeError: Cannot read properties of undefined (reading 'createServerAsync')
-- | -- | -- | --



</body>
</html>

**Comment by jpakusch** [comment]
same here.
The weird thing is that i have 2 iobroker installations. my older, bigger one does not have the issue(running on an AMD CPU in a Proxmox environment)
A new, fresh iobroker instance DOES have the Issue. It is much smaller with less adapters and connections and is running on a raspberry pi 3.

**Comment by Totte23** [comment]
I tried another RESTAPI adapter (https://github.com/ioBroker/ioBroker.rest-api/blob/edit/master/README.md/edit/main/README.md). Same result, same error.
Issue seems to have the root cause somewhere else...?

**Comment by Totte23** [comment]
> I tried another RESTAPI adapter (https://github.com/ioBroker/ioBroker.rest-api?files=1). Same result, same error. Issue seems to have the root cause somewhere else...?

Developer of REST API Adapter just published a Beta (2.0.1). Available via NPM. I just tested this and back in Business…

**Comment by foxriver76** [comment]
fixed with 2.8.0

**Comment by hsteinme** [comment]
Just tested: Fix is successfull. Thank you!
